examples/offline_inference_tt.py

```diff
@@ -119,6 +122,7 @@ def generate_tokens(llm : LLM, prompts, sampling_params, prompt_token_ids=None,
         args.prompts_json,
         measure_perf=args.measure_perf,
         perf_prompt_len=args.perf_prompt_len,
+        perf_output_len=args.perf_output_len,
```

Comment:
Should this line be `max_tokens=args.max_tokens`?

Reply:
Yes, good catch. Updated.